src/sparse_ir/kernel.py

```diff
@@ -184,16 +184,8 @@ def weight_func(self, statistics: str):
         """
         Return the weight function for given statistics.
 
-        This kernel `KFFlat` can be used to represent τ dependence of
-        a bosonic correlation function as follows:
-
-            ∫ KBFlat(x, y) ρ(y) dy
-                == ∫ exp(-Λ * y * (x + 1)/2) / (1 - exp(-Λ*y)) ρ(y) dy
-                == ∫ KFFlat ρ'(y) dy,
-
-        where:
-
-            ρ'(y) == (1/tanh(Λ*y/2)) * ρ(y).
+        Fermion: w(x) = 1
+        Boson: w(y) = tanh(Λ*y/2)
```

Comment:
The comment you deleted is maybe useful ... shall we restore it or otherwise move it to the documentation?

Reply:
OK, I brought back this comment into the doc string.